python/nav/bitvector.py

```diff
@@ -46,7 +49,7 @@ def __setitem__(self, posn, val):
         significant bit is regarded as bit 0 in this context.
         """
         val = val and 1 or 0
-        block = posn / 8
+        block = posn // 8
```

Comment:
I hardly ever need to use integer division and have a very hard coded finger-macro for wrapping the right hand side number in a float. Good thing you're writing this and not me.